contracts/interfaces/I_StarkwareContracts.sol

```diff
@@ -34,13 +34,26 @@ interface I_StarkwareContract {
     *
     * @param  starkKey        The starkKey of the L2 account to deposit into.
     * @param  assetType       The assetType to deposit in.
-    * @param  vaultId         The L2 id to deposit into
-    * @param  quantizedAmount The quantized amount being deposited
+    * @param  vaultId         The L2 id to deposit into.
+    * @param  quantizedAmount The quantized amount being deposited.
     */
     function deposit(
       uint256 starkKey,
       uint256 assetType,
       uint256 vaultId,
       uint256 quantizedAmount
     ) external;
+
+    /**
+    * @notice Register to the Starkware Layer2 Solution.
+    *
+    * @param  ethKey          The ethKey of the L2 account to deposit into.
+    * @param  starkKey        The starkKey of the L2 account to deposit into.
+    * @param  signature       The signature for registering.
+    */
+    function register(
```

Comment:
Where did this function signature come from? I'm not sure if this function exists, I think it should be `registerUser`. You can test using this mock contract: https://github.com/dydxfoundation/governance-contracts/blob/master/contracts/test/MockStarkPerpetual.sol#L49